cfme/common/provider_views.py

```diff
@@ -270,11 +270,19 @@ def is_displayed(self):
     including_entities = View.include(ProviderEntitiesView, use_parent=True)
 
 
+class ContainersProviderEntities(BaseEntitiesView):
+    """
+    represents central view where all QuadIcons, etc are displayed
+    """
+    pass
+
+
 class ContainersProvidersView(ProvidersView):
     """
      represents Main view displaying all Containers providers
     """
     table = Table(locator="//div[@id='list_grid']//table")
+    including_entities = View.include(ContainersProviderEntities, use_parent=True)
```

Comment:
REQUIRED: you don't need this, since ProvidersView already includes ProviderEntities